### ISSUE TITLE
Add button before call "panel:setVisible(false)" will not hide those button

### ISSUE THREAD

**Issue by endlesstravel** [tavuntu/gooi#19]
code like below will not hide button which added to panel
```lua
require("gooi.gooi")

function love.load()
    local pGrid = gooi.newPanel({x = 0, y = 0, w = 100, h = 100, layout = "grid 1x1"})

    pGrid:setVisible(false) -- first , will not work effect.
    
    local btn = gooi.newButton({text = "hide"})
    btn:onRelease(function () pGrid:setVisible(false) end)
    pGrid:add(btn)

    --pGrid:setVisible(false) -- second
    
    gooi.newButton({x = 100, y = 0, w = 100, h = 100, text = "show"}):onRelease(function ()
        pGrid:setVisible(true)
    end)
end

function love.draw()
    gooi.draw()
end

function love.mousereleased(x, y, button) gooi.released() end
function love.mousepressed(x, y, button)  gooi.pressed() end
```
and temporary solution is call `pGrid:setVisible(false)` again after add button

**Comment by gustavostuff** [comment]
This will be fixed in the next version of GOOi (0.0.6), I'm releasing it this week.

**Comment by gustavostuff** [comment]
This should be fixed now, I just released version 0.0.6

**Comment by endlesstravel** [comment]
Great ! Now it is working fine !
